pkg/app/AppService.go

```diff
@@ -465,6 +509,7 @@ func (impl AppServiceImpl) TriggerRelease(overrideRequest *bean.ValuesOverrideRe
 		impl.logger.Errorf("invalid req", "err", err, "req", overrideRequest)
 		return 0, err
 	}
+
```

Comment:
time to break this function in multiple logical function

Reply:
will be done in next release